pkg/controller/controller.go

```diff
@@ -187,13 +190,14 @@ func (c *Controller) ProcessNextPvcItem() bool {
 	}
 	defer c.pvcQueue.Done(key)
 
-	pvc, err := c.pvcFromKey(key)
+	pvc, err := pvcFromKey(c.pvcInformer, key)
 	if err != nil || pvc == nil {
 		return c.forgetKey(key, fmt.Sprintf("ProcessNextPvcItem: error converting key %q to pvc: %v", key, err))
 	}
 
 	// filter pvc and decide if the importer pod should be created
-	createPod, _ := c.checkPVC(pvc, false)
+	//createPod, _ := c.checkPVC(pvc, false)
+	createPod, _ := checkPVC(c.clientset, pvc, false)
```

Comment:
I'm not sure of the reason to convert a method to a func and not export it?  I don't know if we expect a unit test to test `checkPVC` but I guess there's a way to test private funcs by appending _Test_ to the func name...